### ISSUE TITLE
[BUG] [Duke] Shotgun acts like sniper rifle

### ISSUE THREAD

**Issue by axredneck** [ZDoom/Raze#908]
### Raze version

Commit 9618ba1e7875ad4adeb40bfc24d5e17577dac507

### Which game are you running with Raze?

Duke Nukem 3D

### What Operating System are you using?

Linux x86_64

### Please describe your specific OS version

Arch Linux

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

All shotgun pellets hit enemy, even if the enemy is far away. Chaingun is also affected by this bug.
Only pistol and expander should behave this way, but not shotgun nor chaingun.

### Steps to reproduce the behaviour.

Explain how to reproduce
1. New game, L.A. Meltdown
2. Use Give All cheat
3. Select a shotgun
4. Shoot predator trooper from afar

### Your configuration

```ini
https://gist.github.com/axredneck/5e05428758face0dc10bc85f8ceb3517
```

**Comment by coelckers** [comment]
fixed in 5f3065d9f116ca726c020d675038313e31524a6c

**Comment by axredneck** [comment]
Thank You!
